### ISSUE TITLE
Kristal, The (1989)

### ISSUE THREAD

**Issue by TimberlAndRE-de** [exoscoriae/eXoDOS#962]
wrong version  it's for amiga and atari

**Comment by SmilingSpectre** [comment]
I cannot found PC-specific manual. :(

But on the plus side, I found different Amiga/Atari one, with more background info! Uploaded.

**Comment by TimberlAndRE-de** [comment]
@SmilingSpectre 
i don't think it makes sense to replace the wrong manual with another wrong manual

@exoscoriae i hope that's ok for you, otherwise let me know and i'll upload it to the github

**Comment by SmilingSpectre** [comment]
I think, if we have the wrong manual anyway, then at least it must be _good_. Why have it in the inferior quality, if you can have better? *shrug*

**Comment by TimberlAndRE-de** [comment]
good point, let eXo decide.
uploaded to the github folder

**Comment by TimberlAndRE-de** [comment]
@bigjim161 found a reference card, that proofs @SmilingSpectre is right:
"the current manual is good, on the ref card it's written : "The manual included in your package was written for the Commodore Amiga and Atari ST versions of The Kristal. This card has directions specific to the MS-DOS version.""

@exoscoriae use the manual in the github folder + the reference card (#1416)

**Comment by Eulisker** [comment]
Manual in

- English
- French
- German

**Comment by Eulisker** [comment]
I found the manual of the US Version. The current manual is the european version. As far as i found there are 2 major differences. 
The UK manual has a Story part in the beginning that is missing in the US manual.
The US manual has a walkthrough for the beginning of the game that is missing in the UK manual.

Also the US manual is much nicer (the symbols on the top, the depictions of the game interface) and the IBM Reference card is for the US release.

I would suggest replacing the manual and keeping the UK manual as an extra for the story bits.

Added to the github folder.
